vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_wNa16.py

```diff
@@ -119,14 +127,21 @@ def create_weights(self, layer: torch.nn.Module, input_size: int,
                                                           dtype=torch.int64),
                                          weight_loader=weight_loader)
 
+        # G_IDX (for activation reordering)
```

Comment:
Can you add a test case for this case?
1. tests/quantization/test_compressed_tensors.py
2. add a model to models.txt under tests/weight_loading